api/api.go

```diff
@@ -592,6 +596,60 @@ func (api *Server) StreamLogs(in *iotexapi.StreamLogsRequest, stream iotexapi.AP
 	}
 }
 
+// GetVotes gets votes for req
+func (api *Server) GetVotes(
+	ctx context.Context,
+	in *iotexapi.GetVotesRequest,
+) (*iotexapi.GetVotesResponse, error) {
+	if api.electionCommittee == nil {
+		return nil, nil
+	}
+	height, err := strconv.ParseUint(in.Height, 10, 64)
+	if err != nil {
+		return nil, err
+	}
+	result, err := api.electionCommittee.ResultByHeight(height)
+	if err != nil {
+		return nil, err
+	}
+	name, err := hex.DecodeString(in.Votee)
+	if err != nil {
+		return nil, err
+	}
+	if len(name) != 12 {
```

Comment:
Avoid use hard code number inline

Define 12 in advance

Btw, did you confirm 12 is the rule?

Reply:
Its the same code as https://github.com/iotexproject/iotex-election/blob/master/server/server.go#L266-L297,I'll try add some test